src/Main.ts

```diff
@@ -8,6 +8,10 @@
 /* How to use? : Check the GitHub README
 /* v1.5.6
 /* ----------------------------------------------- */
+declare function require(name: string): any;
+
+require("pathseg");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [require statement not part of an import statement](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5052388)